src/ScottLogic.Internal.Training/IntegrationTests/IntegrationTests.cs

```diff
@@ -0,0 +1,206 @@
+using Microsoft.AspNetCore.Mvc.Testing;
+using System.Net;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using ScottLogic.Internal.Training.Matcher;
+using Xunit;
+
+namespace IntegrationTests
```

Comment:
Good start, but I would check the output of the calls as well, not just the status code and the content type.

The idea behind these tests is that we want to to have test that the parts work together so when we put in matching orders, do the trades get created correctly etc.

Reply:
ok, I will do that.